include/pmacc/mappings/kernel/AreaMapping.hpp

```diff
@@ -75,12 +75,17 @@ namespace pmacc
 
         /** Return index of a supercell to be processed by the given alpaka block
          *
+         * @tparam RelativeToArea Which origin (CORE/BORDER/GUARD) to return supercell index relative to
          * @param blockIdx alpaka block index
          * @return mapped SuperCell index including guards
          */
+        template<uint32_t RelativeToArea = GUARD>
```

Comment:
Could you rename thie to `T_Origin` (and in the comment). That will fit the naming we use in the parts of the code too.

Reply:
Renamed to T_origin